planet/config.ini

```diff
@@ -65,6 +65,9 @@ name = Kestrel Computer Project
 [http://blog.verificationgentleman.com/feeds/posts/default]
 name = Verification Gentleman
 
+[http://abopen.com/blog/category/osddi/feed]
```

Comment:
Ah, that's been on my to-do list for some time. Yes, if you could use HTTP for now and I'll try and fix at some point over the next few weeks. Lots of vhosts on that system and I need to revisit the config.